### DISCUSSION TITLE
[Bug]: I can't import to data object

### DISCUSSION THREAD

**Discussion by just-NPT** [pimcore/pimcore#16201] [Q&A / Support]
### Pimcore version

lastest

### Steps to reproduce

I deployed the pimcore skeleton by docker. I added the configuration of Data Importer and Data Hub in file bundle.php and installed dependencies and bundle in your installation docs in the container supervisor.

### Actual Behavior

When I followed your import tutorial on YouTube, there was only one log that said "Loading source data from configured source..." and nothing got imported. Here some step I followed
![image](https://github.com/pimcore/pimcore/assets/59968811/4f728b43-2cef-4819-8faf-f0eddba2cb14)
![image](https://github.com/pimcore/pimcore/assets/59968811/48308fc5-725d-4766-8770-8770f1d61757)
![image](https://github.com/pimcore/pimcore/assets/59968811/77a95cfc-426a-461e-b010-0e7c66ef62b1)
![image](https://github.com/pimcore/pimcore/assets/59968811/3a10f1ff-a5b4-453f-b8cb-603c8fff0376)
All the mappings are direct
![image](https://github.com/pimcore/pimcore/assets/59968811/33e85929-73a1-4b32-9783-71e8f0033953)
![image](https://github.com/pimcore/pimcore/assets/59968811/aeb45be2-2a95-42de-99d7-8b19f90dc500)
And this is my class object
![image](https://github.com/pimcore/pimcore/assets/59968811/abbf974e-2eb9-4f35-9b84-fbd9eb5db644)

### Expected Behavior

It should import 24 rows of data

**Comment by dvesh3** [comment]
Hi @just-NPT please make sure that you configured the complete execution setup as per https://pimcore.com/docs/platform/Data_Importer/Import_Execution_Details
